streams/src/test/java/org/apache/kafka/streams/processor/internals/HighAvailabilityStreamsPartitionAssignorTest.java

```diff
@@ -131,12 +130,13 @@
 
     // Make sure to complete setting up any mocks (such as TaskManager or AdminClient) before configuring the assignor
     private void configurePartitionAssignorWith(final Map<String, Object> props) {
+        EasyMock.replay(taskManager, adminClient);
+
         final Map<String, Object> configMap = configProps();
         configMap.putAll(props);
 
         streamsConfig = new StreamsConfig(configMap);
         partitionAssignor.configure(configMap);
-        EasyMock.replay(taskManager, adminClient);
```

Comment:
We need to setup the mocks before calling `partitionAssignor.configure()` now, as we call `taskManager#adminClient()` in this method (similar elsewhere)